vector/src/main/java/im/vector/activity/CommonActivityUtils.java

```diff
@@ -493,12 +493,12 @@ public void onSuccess(Void info) {
     }
 
     /**
-     * Start LoginActivity in a new task, and clear any other existing task
+     * Start TchapLoginActivity in a new task, and clear any other existing task
      *
      * @param activity the current Activity
      */
     public static void startLoginActivityNewTask(Activity activity) {
-        Intent intent = new Intent(activity, LoginActivity.class);
+        Intent intent = new Intent(activity, TchapLoginActivity.class);
```

Comment:
Have you check that there is no other usage of LoginActivity in the code?

Reply:
Yes it's done. It was the only use of LoginActivity instead of TchapLoginActivity.